app/views/steps/application/urgent_hearing_details/edit.html.erb

```diff
@@ -1,28 +1,25 @@
 <% title t('.page_title') %>
+<% step_header %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary %>
 
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <%= step_header %>
+    <h1 class="govuk-heading-xl"><%=t '.heading' %></h1>
 
-    <h1 class="heading-xlarge gv-u-heading-xxlarge"><%=t '.heading' %></h1>
+    <%= step_form @form_object, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%= f.govuk_text_area :urgent_hearing_details %>
+      <%= f.govuk_text_field :urgent_hearing_when %>
 
-    <%= step_form @form_object do |f| %>
-      <%= f.text_area :urgent_hearing_details, size: '40x4', class: 'form-control-3-4' %>
-      <%= f.text_field :urgent_hearing_when %>
-
-      <div role="note" aria-label="Information" class="panel panel-border-wide info-notice">
-        <p><%= t '.court_info_html' %></p>
+      <div class="govuk-inset-text">
+        <%= t '.court_info_html' %>
       </div>
 
-      <%=
-        f.radio_button_fieldset :urgent_hearing_short_notice, inline: true do |fieldset|
-          fieldset.radio_input(GenericYesNo::YES, panel_id: :urgent_hearing_short_notice_panel)
-          fieldset.radio_input(GenericYesNo::NO)
-          fieldset.revealing_panel(:urgent_hearing_short_notice_panel) do |panel|
-            panel.text_area :urgent_hearing_short_notice_details, size: '40x4', class: 'form-control-3-4'
-          end
-        end
-      %>
+      <%= f.govuk_radio_buttons_fieldset :urgent_hearing_short_notice, legend: { tag: 'span', size: 'm' } do %>
+        <%= f.govuk_radio_button :urgent_hearing_short_notice, GenericYesNo::YES, link_errors: true %>
+        <%= f.govuk_radio_button :urgent_hearing_short_notice, GenericYesNo::NO do%>
```

Comment:
Sorry I missed this before @Davidslv
The YES radio is the one that opens the text area. Can you change this? It's quite important 😄

Reply:
Fix here: https://github.com/ministryofjustice/c100-application/pull/906